build.gradle

```diff
@@ -31,18 +31,19 @@ dependencies {
     testImplementation 'com.tngtech.archunit:archunit-junit4:' + ARCHUNIT_VERSION
 
     intellijPlatform {
-        intellijIdeaCommunity('2024.2.1')
+        intellijIdeaCommunity('2025.1.1')
         pluginVerifier()
         bundledPlugin("com.intellij.java")
         bundledPlugin("org.jetbrains.idea.maven")
+        bundledPlugin("Git4Idea")
```

Comment:
Please add to the PR briefly why we need that new dep

Reply:
To be honest, I don't know why we need this.

Reply:
Without it, the plugin does not compile

Reply:
Strange .. but I guess, if we need it, we need it :D